store/tikv/gcworker/gc_worker_test.go

```diff
@@ -30,15 +28,17 @@ import (
 	"github.com/pingcap/kvproto/pkg/errorpb"
 	"github.com/pingcap/kvproto/pkg/kvrpcpb"
 	"github.com/pingcap/kvproto/pkg/metapb"
-	"github.com/pingcap/pd/client"
+	pd "github.com/pingcap/pd/client"
```

Comment:
Why this changed?

Reply:
Because We use this package as `pd.XXX` , and plugin `gofmt` tell me should format like this.
https://github.com/pingcap/tidb/blob/07ef7e43f750e8aeed1015ab38fb7dabeeb4f6bd/store/tikv/gcworker/gc_worker_test.go#L56